ext/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -392,6 +431,17 @@ in the table below.
   If {CL_MUTABLE_DISPATCH_UPDATABLE_FIELDS_KHR} is not specified then it
   defaults to the value returned by the
   {CL_DEVICE_MUTABLE_DISPATCH_CAPABILITIES_KHR} device query.
+
+| {CL_MUTABLE_DISPATCH_ASSERTS_KHR}
+| {cl_mutable_dispatch_asserts_khr_TYPE}
+| This is a bitfield and can be set to a combination of the following values:
+
+  {CL_MUTABLE_DISPATCH_ASSERT_NO_ADDITIONAL_WORK_GROUPS_KHR}
+  An assertion by the user that the number of work-groups of this ND-range kernel will not be updated beyond
+  the number defined when the ND-range kernel was recorded. The number of work-groups is
+  defined as the product for each _i_ from _0_ to _work_dim - 1_ of
+  _ceil((global_work_size[i] - global_work_offset[i])/local_work_size[i])_ (if
+  _global_work_offset_ is NULL, _global_work_offset[i]_ should be replaced with _0_).
```

Comment:
I don't think this is correct - the number of work-groups is only dependent on the global work size and the local work size, and is not dependent on the global work offset.

See: https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#_mapping_work_items_onto_an_nd_range

> The number of work-groups can be computed as:
> 
> (Wx, Wy) = (ceil(Gx / Sx), ceil(Gy / Sy))

Reply:
You are right, my mistake.